netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -258,6 +261,59 @@ _ebpf_sock_addr_get_current_logon_id(_In_ const bpf_sock_addr_t* ctx)
     return logon_id;
 }
 
+static int
+_ebpf_sock_addr_set_redirect_context(_In_ const bpf_sock_addr_t* ctx, _In_ void* data, _In_ uint32_t data_size)
+{
+    int return_value = 0;
+    net_ebpf_sock_addr_t* sock_addr_ctx = CONTAINING_RECORD(ctx, net_ebpf_sock_addr_t, base);
+    void* redirect_context = NULL;
+
+    // Check for invalid parameters.
+    if (data == NULL || data_size == 0) {
```

Comment:
The verifier will likely check for this due to `EBPF_ARGUMENT_TYPE_PTR_TO_READABLE_MEM`. But its good to have redundancy for this. You may resolve this comment.